packages/kbn-optimizer/limits.yml

```diff
@@ -2,8 +2,8 @@ pageLoadAssetSize:
   actions: 20000
```

Comment:
current build error is: 

ERROR Metric overages:
        page load bundle size > unifiedSearch with value of 23016 is greater than the limit of 23000

@ashokaditya I recommend keeping here only changes for your two plugins that were causing issues 👍